components/primitives/FormPanel.tsx

```
@@ -0,0 +1,30 @@
+}
+
+export const FormPanel: React.FC<FormPanelProps> = ({ title, description, onClose, actions, className, children, ...rest }) => {
+  return (
+    <Card elev={1} surface="subtle" className={cn('p-4 space-y-4', className)} {...rest}>
+      <div className="flex items-start justify-between gap-4">
+        <div className="space-y-1">
+          <h4 className="text-lg font-medium text-white tracking-tight">{title}</h4>
+          {description && <p className="text-xs text-[var(--text-color-muted)] leading-relaxed max-w-prose">{description}</p>}
+        </div>
+        {onClose && (
+          <button onClick={onClose} className="text-[11px] text-[var(--text-color-muted)] hover:text-white px-2 py-1 rounded-md hover:bg-white/10">Close</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)